mrp_operations_extension/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+from . import models
+from . import wizard
+from openerp import api, SUPERUSER_ID
+
+
+def create_default_routing_workcenter_line(cr):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        routing_wcs = env['mrp.routing.workcenter'].search(
+            [('op_wc_lines', '=', False)])
+        for routing_wc in routing_wcs:
+            routing_wc.op_wc_lines = [
+                (0, 0, {'workcenter': routing_wc.workcenter_id,
+                        'default': True,
+                        'custom_data': False})]
+
+
+def post_init_hook(cr, pool):
```

Comment:
Can you documment this method to toe README in order to inform users which are checking the documentation that the data structure and the real data will change once this module is installed?.